rules.d/60-gpiochip.rules

```diff
@@ -0,0 +1,19 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="gpiochip_end"
+SUBSYSTEM!="gpio", GOTO="gpiochip_end"
+
+SUBSYSTEMS=="usb", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb --subsystem=usb"
+
+KERNEL!="gpiochip[0-9]*", GOTO="gpiochip_end"
+
+IMPORT{builtin}="path_id"
+ENV{ID_PATH}=="?*",                   SYMLINK+="gpio/by-path/$env{ID_PATH}"
+ENV{ID_PATH_WITH_USB_REVISION}=="?*", SYMLINK+="gpio/by-path/$env{ID_PATH_WITH_USB_REVISION}"
```

Comment:
Please only use WITH_USB_REVISION. ID_PATH is kept only for backward compatibility. Newer rules should use WITH_USB_REVISION.